src/Movie-Summary.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './css/movie-summary.css'
+
+const MovieSummary = (movieToView) => {
+  return (
+    <div className="movie-summary">
+      <img src={movieToView.movieToView.backdrop_path} alt="Backdrop for"/>
+      <h2>{movieToView.movieToView.title}</h2>
```

Comment:
I'm guessing this info will display once we're making the fetch request to the individual movie api endpoint?